sktime/classification/interval_based/_tsf.py

```diff
@@ -97,6 +109,66 @@ def __init__(self,
         # We need to add is-fitted state when inheriting from scikit-learn
         self._is_fitted = False
 
+    @classmethod
```

Comment:
What's the reason for using @classmethod here? To me, it seems like a @staticmethod, since `cls` isn't used inside the function. But perhaps a stand-alone function would be easiest.